src/main/java/it/gov/pagopa/Controller/EnrollmentsController.java

```diff
@@ -0,0 +1,43 @@
+package it.gov.pagopa.Controller;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
```

Comment:
plaese, use [swagger-annotations](https://mvnrepository.com/artifact/io.swagger.core.v3/swagger-annotations) lib instead of `org.eclipse`